vthunder/resource_vthunder_slb_sip_test.go

```diff
@@ -0,0 +1,31 @@
+package vthunder
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+var TEST_SLB_SIP_RESOURCE = `
+resource "vthunder_slb_sip" "testname" {
+
+	sampling_enable {
+	    counters1 = "all"
+	}
+}
+`
+
+//Acceptance test
+func TestSlbSip_create(t *testing.T) {
```

Comment:
Rename the function to start with TestAcc

Reply:
This is already handled.